functions/go/format/README.md

```diff
@@ -0,0 +1,73 @@
+# format
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Format resources in a directory.
```

Comment:
Need to expand the overview, just saying it "Format resources" is tautological

Reply:
Updated.